src/generated/Endpoints.ts

```diff
@@ -2227,11 +2227,19 @@ export interface Endpoints {
   /**
    * @see https://developer.github.com/v3/code-scanning/#get-a-code-scanning-alert
    */
-  "GET /repos/:owner/:repo/code-scanning/alerts/:alert_id": {
+  "GET /repos/:owner/:repo/code-scanning/alerts/:alert_number": {
```

Comment:
This is a breaking change, we should support both.

Reply:
What is complicated here is that we need to differentiate between deprecated parameter names that URL parameters or not. If they are, we have to create a new type for the path, and replace the new parameter name with the old in that new type. All other types of deprecated parameters can just be added.

Reply:
resolved via https://github.com/octokit/types.ts/pull/165/commits/4bc46d64abb091d38b8d203cd61cbfbdaa92cb73